src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleDeclarations = sourceString
+    .split(';')
+    .map((string) => string.trim().split(':'));
+
+  const trimmedStyleDeclarations = styleDeclarations
+    .filter((innerArr) => innerArr.length === 2)
+    .map((element) => element.map((item) => item.trim()));
+
+  const styleObject = trimmedStyleDeclarations.reduce((acc, arrayElement) => {
```

Comment:
`arrayElement` is a bad name that doesn't describe at all

Reply:
ok, I'll rename it `style` since it's an array of arrays of styles.